eventarc/audit-storage/src/main/java/com/example/cloudrun/EventController.java

```diff
@@ -53,7 +52,7 @@
     }
 
     String ceSubject = headers.get("ce-subject");
-    String msg = String.format("GCS CloudEvent type: %s.", ceSubject);
+    String msg = String.format("Detected change in GCS bucket: %s.", ceSubject);
```

Comment:
```suggestion
    String msg = "Detected change in GCS bucket: " + ceSubject;
```
String.format has poor performance

Reply:
Fixed, thanks.